opengovsg/plumber
# PLU-125: [EXCEL-8] Get cell value action

## Problem
Users want to read data from their excel file.

## Solution
This implements a simple "get cell value" action that allows for getting values of up to **3 cells**.

At some point, we will need something like a "get first table row matching some criteria" action, but these advanced actions will later; it's not straightforward to implement without bashing our rate limits.

## Tests
- Check that I can read data from specified cells
- Check for read what you write consistency - put this action after a "create table row" action, and check that I can read what was written, and that formula output is also consistent.
